framework/src/components/logger/logger.js

```diff
@@ -26,6 +27,55 @@ const createDirIfNotExist = filePath => {
 	fs.mkdirSync(dir, { recursive: true });
 };
 
+// Levels
+const colors = {
+	reset: '\x1b[0m',
+	red: '\x1b[31m',
+	green: '\x1b[32m',
+	yellow: '\x1b[33m',
+	blue: '\x1b[34m',
+	meganta: '\x1b[35m',
+	cyan: '\x1b[36m',
+	white: '\x1b[37m',
+};
+
+const setColor = (color, str) => `${colors[color]}${str}${colors.reset}`;
+
+const levelToName = {
+	10: setColor('yellow', 'TRACE'),
+	20: setColor('meganta', 'DEBUG'),
+	30: setColor('cyan', 'INFO'),
+	40: setColor('yellow', 'WARN'),
+	50: setColor('red', 'ERROR'),
+	60: setColor('red', 'FATAL'),
+};
+
+class ConsoleLog {
+	// eslint-disable-next-line
+	write(rec) {
+		try {
+			let log = util.format(
+				'%s [%s] %s %s (module=%s)\n',
+				new Date(rec.time).toISOString(),
```

Comment:
```suggestion
				new Date(rec.time).toLocaleTimeString('en-US', { hour12: false }),
```

Reply:
I think it's more closer to the bunyan, but for log, i think it's better to have the date too

Reply:
For a certain session, it will just be the same date repeating every line.